styles.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700,800,900&display=swap');
+*
+{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+}

```

Comment:
Add an empty line after each class or tag is styled.

Reply:
Done